Jonathan/restaurants_challenge.py

```diff
@@ -0,0 +1,93 @@
+print("Challenge: Favourite Restaurants")
+
+print()
+
+print("Question 1")
+
+'''
+Below is a dictionary consisting of details of 1 restaurant fetched from Yelp.
+
+Go through the dictionary and print out the following 3 pieces of information about the restaurant:
+1. The latitude and longitude of Four Barrel Coffee
+2. The complete address of Four Barrel Coffee, formatted as a string - it should include the address, city, state and the zip code.
+3. The website of Four Barrel Coffee
+'''
+
+
+restaurant = {
+    "name": "Four Barrel Coffee",
+    "url": "https://www.yelp.com/biz/four-barrel-coffee-san-francisco",
+    "latitude": 37.7670169511878,
+    "longitude": -122.42184275,
+    "city": "San Francisco",
+    "country": "US",
+    "address2": "",
+    "address3": "",
+    "state": "CA",
+    "address1": "375 Valencia St",
+    "zip_code": "94103",
+    "distance": 1604.23,
+    "transactions": ["pickup", "delivery"]
+}
+
+print(restaurant)
+
+# TODO: Write code to print the latitude and longitude of Four Barrel Coffee.
+# TODO: Write code to print the complete address of the Four Barrel Coffee, formatted as a string - it should include the address, city, state and the zip code.
+# TODO: Write code to print the URL of the website of Four Barrel Coffee.
+
+
+print(f'The latitude is {resturants["latitude"]}')
+print(f'The longitude is {resturants["latitude]}')
```

Comment:
Take a look at line 41. Notice any mistake?

Reply:
i fixed it